src/components/UserProjectsPage/index.jsx

```diff
@@ -214,7 +214,7 @@ class UserProjectsPage extends React.Component {
             <div className="ModalFormInputs">
               <Select
                 required
-                placeholder="Choose a location"
+                placeholder="Choose Datacenter location"
```

Comment:
Can we have `Datacenter` in lowercase. Also, do we still need to have `location` there? how about just `data center`?